docs/reference/commandline/dockerd.md

```diff
@@ -1172,6 +1173,9 @@ set this parameter separately for each daemon.
 - `-p, --pidfile=/var/run/docker.pid` is the path where the process ID of the daemon is stored. Specify the path for your
 pid file here.
 - `--host=[]` specifies where the Docker daemon will listen for client connections. If unspecified, it defaults to `/var/run/docker.sock`.
+- `--iptables=false` prevents the Docker daemon from adding iptables rules. If
+  multiple daemons manage iptables rules, they may overwrite rules set by
+  another daemon
```

Comment:
@aaronlehmann 
Not strictly, it depends on how they want to use the containers. They can still run containers on the bridge networks just that they will have certain connectivity limitations. As @thaJeztah said, if they disable this they need to manage the iptables themselves and they are likey aware of the limitations.